### DISCUSSION TITLE
Some feature suggestions

### DISCUSSION THREAD

**Discussion by uwuclxdy** [tribalfs/GalaxyMaxHzPub#135] [Ideas]
Hello there! I saw a recent update and I figured why not give you some ideas or suggestions for the next one ;). Appreciate your work so far, these are the things I miss:

- being able to change charging limit from notification or at least open the menu
- persistent charging notification (impossible to clear while device is charging)
- Net speed quick settings shortcut
- battery temp overlay while charging with screen off (customizable position, size and font). Also maybe other togglable parameters like charging wattage and charging limit (each in a separate line, for example:
"Battery: x°C"
"xW"
"Charge limit: x%")
I think the majority of the code for the last feature could just be copied from refresh rate overlay, but I can't say.

Its already an awesome app and it's fine if these things can't be added. I didn't find any app that would have this functionality, and it could be another reason for people to buy premium version.

**Comment by weaster007** [comment]
I would also really appreachiate an indicator for the discharge (for example -500mA) or charging (e.g. +2000mA)  just like the network speed in the status bar. 
Really cool app. Enjoying it!

**Comment by Princeofpersiattt** [comment]
> Hello there! I saw a recent update and I figured why not give you some ideas or suggestions for the next one ;). Appreciate your work so far, these are the things I miss:
> 
> * being able to change charging limit from notification or at least open the menu
> * persistent charging notification (impossible to clear while device is charging)
> * Net speed quick settings shortcut
> * battery temp overlay while charging with screen off (customizable position, size and font). Also maybe other togglable parameters like charging wattage and charging limit (each in a separate line, for example:
>   "Battery: x°C"
>   "xW"
>   "Charge limit: x%")
>   I think the majority of the code for the last feature could just be copied from refresh rate overlay, but I can't say.
> 
> Its already an awesome app and it's fine if these things can't be added. I didn't find any app that would have this functionality, and it could be another reason for people to buy premium version.

Interesting suggestions. Some of them were added but I hope more of  these are added. Specially battery info while screen off.
